src/wp-content/themes/twentytwentyone/assets/sass/04-elements/blockquote.scss

```diff
@@ -57,8 +62,6 @@ blockquote {
 
 	&:before {
 		content: "\201C";
-		font-size: var(--quote--font-size);
```

Comment:
I removed this to make the quote symbol size be relative to the font size because it did not look well when the font size was either very small or very large.